app/controllers/standup_meetings/completions_controller.rb

```diff
@@ -18,6 +18,6 @@ def create
 
   def standup_meeting_params
     params.require(:standup_meeting).permit(:meeting_date, :yesterday_work_description, :today_work_description,
-      :blockers_description, :status)
+      :blockers_description, :status, :yesterdays_description, :todays_description, :blockers)
```

Comment:
how come we have both of these now? Seems like too many params duplicated?

Reply:
Good point, fixed!